test/ssc_test/cmod_tcsmolten_salt_test.cpp

```diff
@@ -428,7 +428,7 @@ TEST(Mspt_cmod_csp_tower_eqns, Case6) {
 	vd->assign("p_ref", 115.);
 	vd->assign("design_eff", 0.412);
 	vd->assign("tshours", 10.);
-	vd->assign("demand_var", NULL);
```

Comment:
How was NULL processed?

Reply:
I don't know. Perhaps it runs alright on Windows, but was getting an implicit conversion error for me due to there being several constructors that could be used for NULL. As the var is type SSC_NUMBER I switched it to 0.

@Matthew-Boyd